packages/ra-core/src/util/hooks.ts

```diff
@@ -1,33 +1,8 @@
-import * as React from 'react';
-import { useState, useRef, useEffect, useCallback } from 'react';
+import { useRef, useEffect, useState } from 'react';
 import isEqual from 'lodash/isEqual';
 
 // thanks Kent C Dodds for the following helpers
 
-export function useSafeSetState<T>(
```

Comment:
Please keep the export for backward compatibility, even if we don't use it. Mark it as deprecated instead.

Reply:
bump

Reply:
useSafeSetState is back